content/en/apps/guides/hosting/ec2-setup-guide.md

```diff
@@ -1,110 +1,124 @@
 ---
-title: "EC2 Setup"
-linkTitle: "EC2 Setup"
-weight: 
+title: "EC2 Hosting"
```

Comment:
Should this be "AWS Hosting"? I suspect more people know what AWS is than EC2.

Reply:
good call. fixed in c47fdb7